test/Microsoft.Framework.Runtime.FunctionalTests/ServiceBreadcrumbsFacts.cs

```diff
@@ -31,7 +31,6 @@ public void BreadcrumbsAreCreatedSuccessfully()
             breadcrumbs.AddBreadcrumb("Test", new NuGet.SemanticVersion("1.0.0"));
             breadcrumbs.WriteAllBreadcrumbs();
 
-            Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test")));
             Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test.1.0.0")));
```

Comment:
Unfortunately, that's not easy... It could mess up with the files on the user's machine plus it would require admin rights to cleanup. I did a manual verification to see that the breadcrumb is written.